.circleci/config.yml

```diff
@@ -0,0 +1,471 @@
+                    - rocketchip-{{ .Branch }}-{{ .Revision }}
+
+            - run:
+                name: Run rocketchip benchmark tests
+                command: make run-bmark-tests -C sims/verisim SUB_PROJECT=rocketchip
+
+#    hwacha-run-benchmark-tests:
+#        docker:
+#            - image: riscvboom/riscvboom-images:0.0.5
+#        environment:
+#            JVM_OPTS: -Xmx3200m # Customize the JVM maximum heap limit
+#            TERM: dumb
+#
+#        steps:
+#            # Checkout the code
+#            - checkout
+#
+#            - run:
+#                name: Create hash of toolchains
+#                command: |
+#                    .circleci/create-hash.sh
+#
+#            - restore_cache:
+#                keys:
+#                    - riscv-tools-installed-v1-{{ checksum "../riscv-tools.hash" }}
+#
+#            - restore_cache:
+#                keys:
+#                    - hwacha-{{ .Branch }}-{{ .Revision }}
+#
+#            - run:
+#                name: Run hwacha benchmark tests
+#                command: make run-bmark-tests -C sims/verisim SUB_PROJECT=hwacha
+
+# Order and dependencies of jobs to run
+workflows:
+    version: 2
+    build-and-test-rebar-integration:
+        jobs:
+            # Make the toolchains
+            - install-riscv-toolchain
+
+            - install-esp-toolchain
+
+            # Build verilator
+            - install-verilator
+
+            # Prepare the verilator builds
+            - prepare-example:
+                requires:
+                    - install-riscv-toolchain
+                    - install-verilator
+
+            - prepare-boomexample:
+                requires:
+                    - install-riscv-toolchain
+                    - install-verilator
+
+            - prepare-boom:
+                requires:
+                    - install-riscv-toolchain
+                    - install-verilator
+
+            - prepare-rocketchip:
+                requires:
+                    - install-riscv-toolchain
+                    - install-verilator
+
+            - prepare-hwacha-verilog-only:
+                requires:
+                    - install-riscv-toolchain
+                    - install-verilator
+
+            # Run the respective tests
+
+            # Run the example tests
+            - example-run-benchmark-tests:
+                requires:
+                    - install-riscv-toolchain
+                    - prepare-example
+
+            - boomexample-run-benchmark-tests:
+                requires:
+                    - install-riscv-toolchain
+                    - prepare-boomexample
+
+            - boom-run-benchmark-tests:
+                requires:
+                    - install-riscv-toolchain
+                    - prepare-boom
+
+            - rocketchip-run-benchmark-tests:
+                requires:
+                    - install-riscv-toolchain
+                    - prepare-rocketchip
+
+#            - hwacha-run-benchmark-tests:
```

Comment:
This is a non-issue since this doesn't work now, but hwacha should depend on `install-esp-toolchain`.